### PR TITLE
update callback workflow

### PR DESCRIPTION
**Pull request by uday31in**
**Proposed changes**:
- Schema validation for callback workflow
- reanme the UploadBlob-Artifact from .json to .zip

### CODE REVIEW

**Comment by uday31in** [comment]
image

File: utilities/logicApp/Callback-Workflow/workflow.json [hunk 1]
```diff
@@ -344,7 +344,14 @@
...
                     "schema": {
                         "type": "object",
-                        "properties": {}
+                        "properties": {
+                            "id": {
+                                "type": "string"
+                            },
+                            "state": {
+                                "type": "string"
+                            }
```
Comment (marvinbuss) [inline utilities/logicApp/Callback-Workflow/workflow.json:353]: Please remove as these properties are not included in the body. This only validates the body.

**Review by marvinbuss — CHANGES_REQUESTED** [review]
Please see below.

File: utilities/logicApp/Callback-Workflow/workflow.json [hunk 2]
```diff
@@ -344,7 +344,14 @@
...
+                        }
```
Comment (marvinbuss) [inline utilities/logicApp/Callback-Workflow/workflow.json:354]:
```suggestion
                        "properties": {}
```
Please revert.

**Review by marvinbuss — CHANGES_REQUESTED** [review]
Please just revert.

**Review by marvinbuss — APPROVED** [review]
LGTM!

**Pull request merged**